core/arch/arm/plat-rockchip/conf.mk

```diff
@@ -10,6 +10,7 @@ CFG_SHMEM_START ?= 0x68600000
 CFG_SHMEM_SIZE ?= 0x00100000
 endif
 
+$(call force,CFG_DT,y)
```

Comment:
This makes it impossible to turn off `CFG_DT`, is it really what you want? It seems to me that `CFG_DT ?= y` would be more appropriate, because some downstream projects may still want to build without DT support.